source/unified-test-format/tests/valid-pass/initialCollectionData-collectionOptions.yml

```diff
@@ -0,0 +1,39 @@
+description: initialCollectionData-collectionOptions.yml
+
+schemaVersion: '1.5'
+
+createEntities:
+  - client:
+      id: &client0 client0
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name database0
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+
+initialData:
+  - collectionName: *collection0Name
+    databaseName: *database0Name
+    collectionOptions:
+      capped: true
+      size: &cappedSize 512
+    documents:
+      - { _id: 1, x: 11 }
+
+tests:
+  - description: collection is created with the correct options
+    operations:
+      # Execute a collStats command to ensure the collection was created with the correct options.
+      - name: runCommand
+        object: *database0
+        arguments:
+          commandName: collStats
```

Comment:
I know #944 was closed and moved internally for the time being, but heads up to @patrickfreed in case this test will need to be skipped for serverless. IIRC, `$collStats` isn't supported but I'm not sure about the generic command.

Perhaps there's a more general concern with tests running arbitrary commands that might not be in the versioned API, which will required auditing during the course of that project.

Reply:
Thanks for the heads up. I made a note to investigate this and skip any of the tests added in this PR or in #960 if they're incompatible with serverless.

Reply:
Bumping this previously resolved comment.

[Unsupported Commands in Serverless Instances](https://www.mongodb.com/docs/atlas/unsupported-commands/#std-label-atlas-serverless-command-limitations) doesn't include `collStats` so I think we're fine here. Just a reminder to run these tests through Atlas Serverless if you get a chance so we double-check before merging. Feel free to resolve after doing so.

Reply:
Not required by [serverless testing](https://github.com/mongodb/specifications/tree/master/source/serverless-testing#existing-spec-tests), so totally up to you.

I don't think the page I linked above existed when this PR was originally opened, so I just wanted to circle back and cloe the loop on this thread in case anyone comes across it down the line.